Panels/SongEditor/SongEditor.gd

```diff
@@ -1,14 +1,52 @@
 extends VBoxContainer
 
-var InstrumentTrack = preload("./InstrumentTrack.tscn")
+var track_name = preload("./TrackName.tscn")
+var song_segment = preload("./SongSegment.tscn")
```

Comment:
This line seems to be extra, the file's deleted

Reply:
done